modules/ztp-du-host-bios-requirements.adoc

```diff
@@ -1,13 +1,12 @@
-// CNF-1500 ZTP
 // Module included in the following assemblies:
 //
-// scalability_and_performance/ztp-deploying-disconnected.adoc
+// * scalability_and_performance/ztp-far-edge/ztp-reference-cluster-configuration-for-vdu.adoc
 
 :_content-type: PROCEDURE
 [id="configuring_bios_for_du_hosts_{context}"]
-= Configuring BIOS for distributed unit bare-metal hosts
+= Configuring BIOS for low latency and high performance
```

Comment:
You could change "BIOS" to "the BIOS".

Reply:
Is "BIOS" even the correct term anymore? 
As in the "the 90s called they want their technology back"

In the API we refer to "firmware", I believe this is more appropriate

Reply:
Interesting thanks @djuran I'll make this update.